packages/amplify_authenticator/lib/src/widgets/button.dart

```diff
@@ -232,19 +232,25 @@ class ConfirmSignInMFAButton extends AuthenticatorElevatedButton {
 ///
 /// Uses [ButtonResolverKey.signout] for localization
 /// {@endtemplate}
-class SignOutButton extends AuthenticatorElevatedButton {
+class SignOutButton extends StatelessAuthenticatorComponent {
```

Comment:
The changes here don't really have anything to do with this PR. I just noticed while running integ tests that the sign out button was defaulting to occupying the full screen. This is happening because it extends `AuthenticatorElevatedButton`, which is by default full width.

I had changed this to extend `AuthenticatorElevatedButton` when removing amplify theme. These changes revert it back to a normal  material `ElevatedButton`